packages/strapi-design-system/src/BaseButton/BaseButton.tsx

```diff
@@ -36,8 +36,6 @@ export const BaseButton = React.forwardRef<HTMLButtonElement, BaseButtonProps>(
         disabled={disabled}
         padding={2}
         hasRadius
-        background={background}
-        borderColor="neutral200"
```

Comment:
Every component that inherits from `BaseButton` does set these properties explicitly.